vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1807,9 +1807,12 @@ EXPORT(Ptr<void>, sceGxmTextureGetPalette, const SceGxmTexture *texture) {
     return Ptr<void>(texture->palette_addr << 6);
 }
 
-EXPORT(int, sceGxmTextureGetStride, const SceGxmTexture *texture) {
+EXPORT(uint32_t, sceGxmTextureGetStride, const SceGxmTexture *texture) {
     assert(texture != nullptr);
-    return UNIMPLEMENTED();
+    if (texture->texture_type() != SCE_GXM_TEXTURE_LINEAR_STRIDED)
+        return 0;
+
+    return ((texture->mip_filter | (texture->min_filter << 1) | (texture->mip_count << 3) | (texture->lod_bias << 7)) + 1) * 4;
```

Comment:
Since it's complicated and long, I want you to move this into an util in gxm module.

Named gxm::get_stride_in_bytes

Reply:
ah for no calcule value 2 time ok